README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
     basic_auth:
       # Corresponds to your OpenStack User and Project
-      username: <user_id>@<project_id>
+      username: <user_id>|<project_id>
```

Comment:
"@" is already used to qualify the domain (in e-mail addresses). Using IDs is awkward, so we need one symbol to qualify the domain of a username or a project name plus one symbol to separate the user from the authorisation scope.

I thought "|" is a good candidate because it has a connotation of "restricted to".